README.md

```diff
@@ -88,4 +88,4 @@ Kubernetes LDAP is an open source project and contributors are welcome!
 
 ## Licensing
 
-Unless otherwise noted, all code in the Kubernetes LDAP repository is licensed under the [Apache 2.0 license](LICENSE). Some portions of the codebase are derived from other projects under different licenses; the appropriate information can be found in the header of those source files, as applicable.
+Unless otherwise noted, all code in the Kubernetes LDAP repository is licensed under the [Apache 2.0 license](LICENSE). Some portions of the codebase are derived from other projects under different licenses; the appropriate information can be found in the header of those source files, as applicable

```

Comment:
Why remove the period? It is a sentence.

Reply:
i was trying to test trigger the docker build. will revert this change

Reply:
docker will only build on a push of a release tag. If you want ci builds, use Travis. See the .travis.yml in either kapprover or certificate-init-container